js/index.js

```diff
@@ -0,0 +1,122 @@
+const speakers = [
+  {
+    name: 'Yochai Benkler',
+    proffession:
+      'Berkman professor of Entreneurial Legal studies at harvard Law school',
+    description:
+      ' bekler studies commons-based Berkman professor of Entreneurial Legal studies at harvard Law school',
+    photo: '/img/profiles/alex-suprun-ZHvM3XIOHoE-unsplash.jpg',
+    alt: 'Photo by Christopher Campbell on Unsplash',
+  },
+  {
+    name: 'Ali Abbani',
+    proffession:
+      'Berkman professor of Entreneurial Legal studies at harvard Law school',
+    description:
+      ' bekler studies commons-based Berkman professor of Entreneurial Legal studies at harvard Law school',
+    photo: '/img/profiles/nicolas-horn-MTZTGvDsHFY-unsplash.jpg',
+    alt: '',
+  },
+  {
+    name: 'Henry hi',
+    proffession:
+      'Berkman professor of Entreneurial Legal studies at harvard Law school',
+    description:
+      ' bekler studies commons-based Berkman professor of Entreneurial Legal studies at harvard Law school',
+    photo: '/img/profiles/aatik-tasneem-7omHUGhhmZ0-unsplash.jpg',
+    alt: '',
```

Comment:
It is best practice to always have an alt for all your images because we're building the web for everyone, it is called web accessibility a11y